rpc/handlers.go

```diff
@@ -450,3 +453,41 @@ func (h *Handler) Syncing() (*SyncState, *jsonrpc.Error) {
 		},
 	}, nil
 }
+
+func (h *Handler) readState(id *BlockID) (core.StateReader, func() error, error) {
```

Comment:
reading from the state doesnt really happen here
how about something like `stateByBlockID`, to bring it inline with `blockByID`, `blockHeaderByID` etc?

Reply:
Yeah. Just saw your naming in `stateByBlockID`. Will fix

Reply:
Done